tests/test_upload_mission.py

```diff
@@ -7,24 +7,24 @@
 """
 
 import json
+from pathlib import Path
 
 from tests.fixtures import BASE_CONFIG, MODLIST_DICT, MODLIST_JSON
 from zeusops_bot.reforger_config_gen import ReforgerConfigGenerator, extract_mods
 
 
-def test_upload_edits_files(tmp_path):
+def test_upload_edits_files(base_config: Path, mission_dir: Path):
```

Comment:
Ignoring if the actual fixture would be worth it to pack all-in-one, we _could_ make both into a single fixture like `config_info: tuple[Path, Path]` then `base_config, mission_dir = config_info` if you want?

Only useful if we REALLY care about joining the two fixtures because they are really just one fixture split into two variables, which likely isn't the case (and I intentionally don't speak to its validity)